src/core/jpg.js

```diff
@@ -1065,10 +1065,10 @@ var JpegImage = (function JpegImageClosure() {
       var Y, Cb, Cr, k;
       var offset = 0;
       for (var i = 0, length = data.length; i < length; i += 4) {
-        Y = data[i];
-        Cb = data[i + 1];
-        Cr = data[i + 2];
-        k = data[i + 3];
+        Y  = 255-data[i];
```

Comment:
Here, and in other parts of the part, please leave one and only one space around the `-` and `=` operators, e.g., `Y = 255 - data[i]`.